kotest-framework/kotest-framework-multiplatform-plugin-gradle/test-project/.gitignore

```diff
@@ -0,0 +1,9 @@
+
+# lock files are not important in a test project
+yarn.lock
```

Comment:
Is there a reason why we wouldn't keep the lock file? This helps ensure everyone is running the same versions of each dependency.

Reply:
Each test run changed the Kotlin version, which the lock file, so it was constantly changing. Normally yes, it should be committed.

Reply:
Ah, I see, makes sense.